man/linters.Rd

```diff
@@ -84,6 +84,7 @@ The following linters exist:
 \item{\code{\link{paren_body_linter}} (tags: default, readability, style)}
 \item{\code{\link{paren_brace_linter}} (tags: default, readability, style)}
 \item{\code{\link{paste_sep_linter}} (tags: best_practices, consistency)}
+\item{\code{\link{paste_to_string_linter}} (tags: best_practices, consistency)}
```

Comment:
why didn't the tag counts change as well?

Reply:
funky merging... ran again and it updated. confirmed the new counts are right:

```
grep "consistency" inst/lintr/linters.csv | wc -l
# 16
grep "best_practices" inst/lintr/linters.csv | wc -l
# 35
```